modules/accounting/includes/functions/products.php

```diff
@@ -139,11 +153,10 @@ function erp_acct_insert_product( $data ) {
         $wpdb->query( 'COMMIT' );
     } catch ( Exception $e ) {
         $wpdb->query( 'ROLLBACK' );
-
-        return new WP_error( 'product-exception', $e->getMessage() );
+        return new WP_Error( 400,  $e->getMessage() );
```

Comment:
the correct format for sending status code with WP_ERROR is: `return new WP_Error( 'no_posts', __('No post found'), array( 'status' => 404 ) );// status can be changed to any number`